docs/guides/tracking-requests.md

```diff
@@ -34,39 +36,126 @@ This was written for the EvidenceAPI staging database and the steps can be easil
+```sh
+EVIDENCE_API_HOST=$(aws ssm get-parameter --name /evidence-api/production/postgres-hostname --query Parameter.Value)
+EVIDENCE_API_PORT=$(aws ssm get-parameter --name /evidence-api/production/postgres-port --query Parameter.Value)
+JUMP_BOX_NAME=$(aws ssm get-parameter --name /document-evidence-store-production-jump-box-instance-name --query Parameter.Value)
+```
+3. Setup port forwarding which creates a tunnel between your local machine and the jump host. It will map `localhost:EVIDENCE_API_PORT` to `jumpboxhost:EVIDENCE_API_PORT`, so change the localhost if you've got something else running on `port 5502`.
 ```sh
 ssh -4 -i ./private-key.pem -Nf -M -L ${EVIDENCE_API_PORT//\"}:${EVIDENCE_API_HOST//\"}:${EVIDENCE_API_PORT//\"} -o "UserKnownHostsFile=/dev/null" -o "StrictHostKeyChecking=no" -o ProxyCommand="aws ssm start-session --target %h --document AWS-StartSSHSession --parameters portNumber=%p --region=eu-west-2" ec2-user@${JUMP_BOX_NAME//\"}
 ```
-10. Print Evidence API database configuration
+#### Documents API - PRODUCTION
+1. Run the following to create a private key and permission it appropriately
 ```sh
-echo $EVIDENCE_API_PORT
-aws ssm get-parameter --name /evidence-api/staging/postgres-username --query Parameter.Value
-aws ssm get-parameter --name /evidence-api/staging/postgres-password --query Parameter.Value --with-decryption
-aws ssm get-parameter --name /evidence-api/staging/postgres-database --query Parameter.Value
+aws ssm get-parameter --name "/document-evidence-store-production-jump-box-pem-key	" --output text --query Parameter.Value > ./private-key.pem
+chmod 400 ./private-key.pem
 ```
-11. Setup a connection to the database from your favourite SQL tool with the details you just retrieved
+* Note: If you already have the file (from a previous run of this step for example) you will need to remove it before running.
+
+2. Create environment variables by retrieving properties from AWS SSM
+```sh
+DOCUMENTS_API_HOST=$(aws ssm get-parameter --name /documents-api/production/postgres-hostname --query Parameter.Value)
+DOCUMENTS_API_PORT=$(aws ssm get-parameter --name /documents-api/production/postgres-port --query Parameter.Value)
+JUMP_BOX_NAME=$(aws ssm get-parameter --name /document-evidence-store-production-jump-box-instance-name --query Parameter.Value)
+```
+3. Setup port forwarding which creates a tunnel between your local machine and the jump host. It will map `localhost:EVIDENCE_API_PORT` to `jumpboxhost:EVIDENCE_API_PORT`, so change the localhost if you've got something else running on `port 5502`.
+```sh
+ssh -4 -i ./private-key.pem -Nf -M -L ${DOCUMENTS_API_PORT//\"}:${DOCUMENTS_API_HOST//\"}:${DOCUMENTS_API_PORT//\"} -o "UserKnownHostsFile=/dev/null" -o "StrictHostKeyChecking=no" -o ProxyCommand="aws ssm start-session --target %h --document AWS-StartSSHSession --parameters portNumber=%p --region=eu-west-2" ec2-user@${JUMP_BOX_NAME//\"}
+```
+
+#### Connection to the database from your Postgres client
```

Comment:
It might be worth mentioning that this should only need to be done once.

Reply:
Good point! I will also mention that if the localhost ports are changed at anytime, then the db connection config in the client needs to be changed too.